.pre-commit-config.yaml

```
@@ -111,6 +111,6 @@ repos:
         entry: pip-compile -q --strip-extras --no-annotate --output-file=requirements.txt tools/opts.txt setup.cfg --extra docs --extra test --extra yamllint
       - id: pip-compile
         name: pip-compile-upgrade
-        entry: pip-compile -q --strip-extras -q --upgrade --no-annotate --output-file=requirements.txt setup.cfg --extra docs --extra test --extra yamllint
+        entry: pip-compile -q --strip-extras -q --upgrade --no-annotate --output-file=requirements.txt tools/opts.txt setup.cfg --extra docs --extra test --extra yamllint
```

Should this maybe be `lint` instead of `yamllint`?

```suggestion
        entry: pip-compile -q --strip-extras -q --upgrade --no-annotate --output-file=requirements.txt tools/opts.txt setup.cfg --extra docs --extra test --extra lint
```